README.md

```diff
@@ -107,3 +107,30 @@ const refreshToken = getRefreshToken();
 
 - If multiple requests happen simultaneously after token expiration then multiple refresh token requests may be issued.
 - Your backend should allow a few seconds of leeway between when the token expires and when it actually becomes unusable.
+
+## Non-TypeScript implementation
+
+```javascript
+import {useAuthTokenInterceptor} from 'axios-jwt';
+import axios from 'axios';
+
+const apiClient = axios.create();
+
+const requestRefresh = (refresh) => {
+    return new Promise((resolve, reject) => {
+        // notice that this is the global axios instance.  <-- important
+        axios.post('/api/v1/auth/token/refresh/', {
+            refresh
+        })
+            .then(response => {
+                resolve(response.data.accessToken);
+            }, reject);
```

Comment:
sorry dumb question (i've been using `async` for too long and forgot how these work) - what is `reject` doing here?

Reply:
It will reject the outer promise. A better way to understand it coming from async/await is like this:

```javascript
const requestRefresh = async refresh => {
    try {
        const response = await axios.post(/* stuff */)
        return response.data.accessToken;
    } catch (e) { 
        console.error(e);
        console.log('this was rejected');
        throw e;
     }
}
```

Reply:
obviously you'd probably just await the response and return the data and handle let the errors propagate, but; you get the idea.